docs/development/server_administration/docker_swarm/Database.md

```diff
@@ -4,9 +4,9 @@ In order for the new Docker Swarm to serve the Giraf project, the database from
 
 ## Old setup
 
-The old server was configured to use MySql 5.7.11 as can be seen on the picture below.
+The old server was configured to use MySQL 8.0.19 as can be seen on the picture below.
```

Comment:
Not related to the issue per se. Fixed this, because I saw it there, and I knew we updated. There IS an issue in backlog for ensuring consistent documentation in regards to versioning.

Reply:
But this is during the explanation of the "Old setup", so I don't think the version number should be changed

Reply:
You're very right. Corrected!